javascript/core/src/verbs/onehot.ts

```diff
@@ -22,22 +22,29 @@ export interface OnehotArgs extends InputColumnArgs {
  */
 export const onehotStep: ColumnTableStep<OnehotArgs> = (
 	input,
-	{ column, prefix },
+	{ columns, prefix },
 ) => {
-	// note that this ignores potential grouping
-	// TODO: should this only apply to string column types?
-	const distinct = input
-		.rollup({
-			distinct: op.array_agg_distinct(column),
-		})
-		.get('distinct', 0) as any[]
-
-	const args = distinct.sort().reduce((acc, cur) => {
-		acc[prefix ? `${prefix}${cur}` : cur] = escape((d: any) =>
-			d[column] === null ? null : d[column] === cur ? 1 : 0,
-		)
-		return acc
-	}, {} as Record<string, ExprObject>)
+	let args = {}
+
+	columns.forEach(column => {
```

Comment:
This would be cleaner if you used `const args = columns.reduce(...)` instead of reassigning `args` each time

Reply:
I already changed it to the reduce, I don't know why is not updated here.
This is the file in the branch I have the changes to merge: https://github.com/microsoft/data-wrangling-components/blob/feature/onehot-multiple-columns/javascript/core/src/verbs/onehot.ts